### DISCUSSION TITLE
Office_S_Tips/excel/2019/10/10/Excel-Scalanie_danych_z_tabl_przestawej

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#26] [Komentarze]
# Office_S_Tips/excel/2019/10/10/Excel-Scalanie_danych_z_tabl_przestawej

Wielokrotne dane w danej kategorii jako pojedynczy tekst. Także dodatek do Excel 2013- “AK_dodFunkcje.xlam” dla kilku nowych funkcji Excel 2016+ : TEXTJOIN/POŁĄCZ.TEKSTY, CONCAT/ZŁĄCZ.TEKST, SWITCH/PRZEŁĄCZ, IFS/WARUNKI (+ opis szybkiego przełączania języka funkcji Excel)

https://andrzejq.github.io/Office_S_Tips/excel/2019/10/10/Excel-Scalanie_danych_z_tabl_przestawej.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
